### PR TITLE
fix(Opening Invoice Creation Tool): Display message on successful creation of opening invoices

### PR DESCRIPTION
**Pull request by GangaManoj**
_Issue:_
No message is displayed if creation of opening invoices is successful, if there are less than five invoices, causing people to press the button multiple times, thereby creating additional invoices.

_Fix:_
Display appropriate message on successful creation of invoices.

Screenshot 2021-06-10 at 5 58 58 AM

### CODE REVIEW

File: erpnext/accounts/doctype/opening_invoice_creation_tool/opening_invoice_creation_tool.py [hunk 1]
```diff
@@ -213,6 +213,8 @@ def start_import(invoices):
 	if errors:
 		frappe.msgprint(_("You had {} errors while creating opening invoices. Check {} for more details")
 			.format(errors, "<a href='/app/List/Error Log' class='variant-click'>Error Log</a>"), indicator="red", title=_("Error Occured"))
+	if len(invoices) < 5:
```
Comment (deepeshgarg007) [inline erpnext/accounts/doctype/opening_invoice_creation_tool/opening_invoice_creation_tool.py:216]: I guess the screen is already frozen while the invoices are being created. If the invoices are not created via a background job then the invoice names are returned to the client-side so this message can be printed there if invoice names are found

Comment (deepeshgarg007) [inline erpnext/accounts/doctype/opening_invoice_creation_tool/opening_invoice_creation_tool.py]: I guess the screen is already frozen while the invoices are being created. If the invoices are not created via a background job then the invoice names are returned to the client-side so this message can be printed there if invoice names are found

**Review by deepeshgarg007 — CHANGES_REQUESTED** [review]

File: erpnext/accounts/doctype/opening_invoice_creation_tool/opening_invoice_creation_tool.py (hunk 1 above)
Comment (GangaManoj) [reply erpnext/accounts/doctype/opening_invoice_creation_tool/opening_invoice_creation_tool.py]: Done.

File: erpnext/accounts/doctype/opening_invoice_creation_tool/opening_invoice_creation_tool.py [hunk 2]
```diff
@@ -216,7 +216,7 @@ def start_import(invoices):
 	return names
 
 def publish(index, total, doctype):
-	if total < 5: return
+	if total < 50: return
```
Comment (nextchamp-saqib) [inline erpnext/accounts/doctype/opening_invoice_creation_tool/opening_invoice_creation_tool.py]:
```suggestion
	if total < 50:
		return
```

**Review by nextchamp-saqib — CHANGES_REQUESTED** [review]

**Comment by nextchamp-saqib** [comment]
@GangaManoj Port this to `version-13-hotfix`

**Pull request merged**